api/src/main/java/org/pmiops/workbench/google/GoogleRetryHandler.java

```diff
@@ -32,6 +33,10 @@ protected int getResponseCode(Throwable lastException) {
       if (lastException instanceof SocketTimeoutException) {
         return HttpServletResponse.SC_GATEWAY_TIMEOUT;
       }
+      if (lastException instanceof TokenResponseException) {
+        throw ExceptionUtils.codeToException(
```

Comment:
should this be `return` instead of throw?

Reply:
Thanks, copy/paste error... surprisingly it still worked - I'll test this again.